Symfony/CS/Fixer/Symfony/PhpdocVarWithoutNameFixer.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\DocBlock\DocBlock;
+use Symfony\CS\DocBlock\Line;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Graham Campbell <graham@mineuk.com>
+ */
+class PhpdocVarWithoutNameFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens->findGivenKind(T_DOC_COMMENT) as $token) {
+            $doc = new DocBlock($token->getContent());
+
+            // don't process single line docblocks
+            if (1 === count($doc->getLines())) {
+                continue;
+            }
+
+            foreach ($doc->getAnnotationsOfType(array('var', 'type')) as $annotation) {
+                $this->fixLine($doc->getLine($annotation->getStart()));
+            }
+
+            $token->setContent($doc->getContent());
```

Comment:
You can early skip iteration if there are no `var` annotation.
Then there will be no nee for regenerating content ;)